doc/Background.md

```diff
@@ -0,0 +1,44 @@
+# Other Background Material
+
+## Quick links
+
+## Other reading materials
+
+- Computer-readable legislation project, main page: <https://osf.io/yzf6x/>
+- Meng's thoughts on L4 syntax: <https://docs.google.com/spreadsheets/d/1RW5wLksZY7BBpjkbYzHDAjAgan994nnTdZiqhTFxoJw/edit?gid=1732775477#gid=1732775477>
```

Comment:
I'd recommend using markdown syntax for this like
```ms
[description of the linnk](https://example.com/foo/bar)
```

Reply:
ok, reformatted